pkgs/development/python-modules/jmp/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, jax
+, jaxlib
+}:
+
+buildPythonPackage rec {
+  pname = "jmp";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "deepmind";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-t78zg6H17l/yBPxYygFjPRdclr88BoTDhPUhsEcNUM0=";
+  };
+
+  propagatedBuildInputs = [
+    jax
+    jaxlib
+  ];
```

Comment:
Could check the imputs? Upstream's [`requirements.txt`](https://github.com/deepmind/jmp/blob/main/requirements.txt) has additional run-time requirements.